path_login.go

```diff
@@ -277,6 +203,120 @@ func (b *jwtAuthBackend) pathLoginRenew(ctx context.Context, req *logical.Reques
 	return resp, nil
 }
 
+func (b *jwtAuthBackend) verifyToken(ctx context.Context, config *jwtConfig, role *jwtRole, rawToken string) (map[string]interface{}, error) {
```

Comment:
To distinguish this from the jwt signing verification method could we rename to verifyOIDCToken?

Reply:
Renamed